astroquery/jplhorizons/core.py

```diff
@@ -973,19 +973,26 @@ def vectors_async(self, get_query_payload=False,
 
         # configure request_payload for ephemerides query
         request_payload = OrderedDict([
-            ('batch', 1),
-            ('TABLE_TYPE', 'VECTORS'),
+            ('format', 'text'),
+            ('EPHEM_TYPE', 'VECTORS'),
             ('OUT_UNITS', 'AU-D'),
             ('COMMAND', '"' + commandline + '"'),
             ('CENTER', ("'" + str(self.location) + "'")),
             ('CSV_FORMAT', ('"YES"')),
-            ('REF_PLANE', {'ecliptic': 'ECLIPTIC', 'earth': 'FRAME',
-                           'body': "'BODY EQUATOR'"}[refplane]),
-            ('REF_SYSTEM', 'J2000'),
+            ('REF_PLANE', {
+                'ecliptic': 'ECLIPTIC',
+                'earth': 'FRAME',
+                'frame': 'FRAME',
+                'body': "'BODY EQUATOR'"
+            }[refplane]),
```

Comment:
YMMV, but I don't find this type of formatting readable, it totally loses the structure info that was in the previous that REF_PLANE and below VECT_CORR have a set of values options

Reply:
I thought this was more readable, but I'm happy to change it back.

Reply:
no big deal, for me it's not about the line breaks, but the indents. They directing my eyes straight away to see where the structure boundaries are without the need of counting of closing brackets and parens left on their own in a new line.

```
            ('VEC_CORR', {'geometric': '"NONE"',
                          'astrometric': '"LT"',
                          'apparent': '"LT+S"'}[aberrations]),
```

Reply:
(That being said, you're the maintainer of this module, so either way is fine by me.)

Reply:
You have a fair point 👍🏻